drivers/gpio/gpio_mcp23x17.h

```diff
@@ -46,28 +46,54 @@ extern "C" {
 #define MCP23X17_ADDR                   0x40
 #define MCP23X17_READBIT                0x01
 
+/** Store the port 0/1 data for each register pair. */
```

Comment:
You won't need this after fixing the way the I/O is done.

Reply:
Not sure why I won't need these?
Please elaborate.

Reply:
I think that's clear now: `union gpio_mcp23x17_port_data` became unnecessary with the refactoring.